index.html

```diff
@@ -14,6 +14,59 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+        
+        <img src="./img\karma-logo.svg" alt="karma-logo" id="logo"/>
```

Comment:
```suggestion
        <img src="./img/karma-logo.svg" alt="Karma logo" id="logo"/>
```
We shouldn't mix up the slashes for the file path. Also, the `alt` description is a sentence, not code, so we shouldn't use the hyphen instead of a space